jujugui/static/gui/src/app/app.js

```diff
@@ -1051,39 +1040,47 @@ YUI.add('juju-gui', function(Y) {
 
     _emptySectionC: function() {
       // If the model name has been hidden by the profile then show it again.
-      document.getElementById(
-        'environment-switcher').classList.remove('hidden');
+      this._renderBreadcrumb({ showEnvSwitcher: true });
       ReactDOM.unmountComponentAtNode(
         document.getElementById('charmbrowser-container'));
     },
 
     /**
-      Renders the environment switcher
+      Renders the breadcrumb component to the DOM.
 
-      @method _renderEnvSwitcher
+      @method _renderBreadcrumb
+      @param {Object} options
+        showEnvSwitcher: false
     */
-    _renderEnvSwitcher: function() {
-      if(this.env.findFacadeVersion('ModelManager') === null &&
+    _renderBreadcrumb: function(
+        { showEnvSwitcher=true, authEndpoint='charmstore' } = {}) {
```

Comment:
While a little ugly this is the syntax to enable 'named' params in JS without any 'required' params.